browser/components/CodeEditor.js

```diff
@@ -344,7 +377,12 @@ export default class CodeEditor extends React.Component {
       inputStyle: 'textarea',
       dragDrop: false,
       foldGutter: true,
-      gutters: ['CodeMirror-linenumbers', 'CodeMirror-foldgutter'],
+      lint: {
+        'getAnnotations': validatorOfMarkdown,
+        'async': true
+      },
+      mode: 'markdown',
```

Comment:
Please don't use `markdown` mode here. We set the mode according to the props or self-detecting language result if the CodeEditor is used in snippet note.
https://github.com/BoostIO/Boostnote/blob/885f656d3458af090c62f81b4f8a1c775effcdc6/browser/components/CodeEditor.js#L357-L361

Reply:
I understood.
I fixed it. Please check it again.